torch/_meta_registrations.py

```diff
@@ -2639,6 +2639,21 @@ def meta_bucketize(self, boundaries, *, out_int32=False, right=False):
     ).contiguous()
 
 
+@register_meta(aten._upsample_bilinear2d_aa.default)
```

Comment:
So what's going on, are you doing this in Python or in C++ lol. You don't need both.

Reply:
I think the logic in UpSampleBilinear2d.cpp is not a Meta kernel, instead it looks more like a C++ binded to Python API. Line 180 of UpSampleBilinera2d.cpp invokes this Python Meta kernel. PTAL.

Reply:
I see, there's a vec and non-vec overload, ok